test_ceph_daemon.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -ex
+
+fsid=2a833e3f-53e4-49a7-a7a0-bd89d193ab62
+#image=ceph/daemon-base:latest-master-devel
+image=cephci/daemon-base:wip-ceph-daemon-4c73385-centos-7-x86_64-devel
```

Comment:
cleanup later?

Reply:
yeah
once this is in master i think i'll make the default the current dev master.. until octopus freeze happens, at which point we can change it to that.  none of this works with nautilus